android/tangram/src/com/mapzen/tangram/geometry/Geometry.java

```diff
@@ -0,0 +1,35 @@
+package com.mapzen.tangram.geometry;
+
+import java.util.Map;
+
+/**
+ * Geometry is an abstract container of LngLat points and associated properties
+ */
+public abstract class Geometry {
+
+    protected double[] coordinates;
+    protected int[] rings;
+    protected String[] properties;
+
+    public double[] getCoordinateArray() {
```

Comment:
Turns out we need these public to use them from the `com.mapzen.tangram` package - instead I've added comments that they are not for public use.